.github/workflows/wheels.yml

```diff
@@ -63,7 +63,7 @@ jobs:
       # https://github.com/actions/download-artifact
       # These actions should remain @v3 for now, ignore dependabot proposed upgrade.
       # https://github.com/actions/upload-artifact/issues/478
```

Comment:
This issue mentioned here (https://github.com/actions/upload-artifact/issues/478) is resolved. It should be safe to use @v4. We should also bump the version of upload-artifact action to @v4.